modules/javafx.graphics/src/main/java/com/sun/marlin/MarlinProperties.java

```diff
@@ -189,6 +202,10 @@ public static float getSubdividerMinLength() {
         return getFloat("prism.marlin.clip.subdivider.minLength", 100.0f, Float.NEGATIVE_INFINITY, Float.POSITIVE_INFINITY);
     }
 
+    public static boolean isUseDPQS() {
+        return getBoolean("prism.marlin.useDPQS", "true");
```

Comment:
In order to reduce risk, it might make sense to disable DPQS by default initially. It would then be available to anyone who wants to try it out with more complex paths, but would be less prone to possible regressions. What do you think?